From: Ray Johnston <ray.johnston@artifex.com>
Date: Mon, 14 Dec 2020 16:39:50 +0000 (-0800)
Subject: Fix bug 703270: Wrong path for PostScript helper file in ps2epsi
X-Git-Tag: ghostpdl-9.54.0-test-base-0~13
X-Git-Url: http://git.ghostscript.com/?p=ghostpdl.git;a=commitdiff_plain;h=c6166768c6e963b0fe28ccdb266629443e521381

Fix bug 703270: Wrong path for PostScript helper file in ps2epsi

In the change mentioned in the bug, rather than rely on the LIBPATH
search method, the ps2epsi script assumed that pd2epsi.ps would be
in the same directory as the 'gs' executable, which is not correct.

Change to use bare 'ps2epsi.ps' so that it will be found on the
LIBPATH as instialled by: make install
---

diff --git a/lib/ps2epsi b/lib/ps2epsi
index 7590cb5a7..dbfc9fb3d 100755
--- a/lib/ps2epsi
+++ b/lib/ps2epsi
@@ -40,8 +40,8 @@ else
 	outfile=$2
 fi
 
-# Note, we expect 'ps2epsi.ps' to be in the same directory as 'ps2epsi'
+# Note, we expect 'ps2epsi.ps' to be on one of the search paths which can be seen by: gs -h
 "$GS_EXECUTABLE" -q -dNOOUTERSAVE -dNODISPLAY -dLastPage=1 -sOutputFile="${outfile}" \
-		--permit-file-all="${infile}" -- "$LIBDIR/ps2epsi.ps"  "${infile}" 1>&2
+		--permit-file-all="${infile}" -- ps2epsi.ps  "${infile}" 1>&2
 
 exit 0
